lib/Github/Client.php

```diff
@@ -143,6 +145,11 @@ public function api($name)
                 $api = new Api\Markdown($this);
                 break;
 
+            case 'notification':
+            case 'notifications':
+                $api = new Api\Activity\Notification($this);
```

Comment:
This is wrong. If it is in the ``Activity`` subnamespace, it should be acess from an ``Api\Activity`` API, not from the main client, to be consistent with the current architecture of the library